arc-api/server/models/subscription.js

```diff
@@ -192,6 +206,11 @@ module.exports = function(Subscription) {
       },
       json: true
     }, function(err, res, body) {
+      if (err && mode !== 'online') {
```

Comment:
Can we reverse this? We wouldn't normally send `mode: 'online'` only real use case for us is `mode: 'offline'`